protocol/kafka_confluent/v2/protocol.go

```diff
@@ -231,15 +230,18 @@ func (p *Protocol) Receive(ctx context.Context) (binding.Message, error) {
 func (p *Protocol) Close(ctx context.Context) error {
 	p.closerMux.Lock()
 	defer p.closerMux.Unlock()
+	logger := cecontext.LoggerFrom(ctx)
 
 	if p.consumerCancel != nil {
 		p.consumerCancel()
 	}
 
 	if p.producer != nil && !p.producer.IsClosed() {
+		// Flush and close the producer and the events channel
+		for p.producer.Flush(10000) > 0 {
+			logger.Info("Still waiting to flush outstanding messages")
```

Comment:
```suggestion
			logger.Info("Flushing outstanding messages")
```

Question: `10000` is 10 seconds timeout or delay? How often is this message printed? Risk of floding logs?